README.win32.adoc

```diff
@@ -26,10 +26,12 @@ Here is a summary of the main differences between these ports:
 |=====
 
 [[tb1]]
-(*):: Cygwin-generated `.exe` files refer to a DLL that is distributed under the
-GPL.  Thus, these `.exe` files can only be distributed under a license that is
-compatible with the GPL.  Executables generated by Microsoft Visual C++ or
-Mingw-w64 have no such restrictions.
+(*):: Cygwin-generated `.exe` files are linked with the Cygwin DLL. Since
+version 2.5.2 released in June 2016, this DLL is licensed under the LGPLv3 with
+a static linking exception; prior versions of the DLL are distributed under the
+GPL. Thus, these `.exe` files can only be distributed under compatible license.
```

Comment:
You should clarify that the "can only be distributed" sentence only concerns the older version case.